src/pages/ProductsPage.jsx

```diff
@@ -1,7 +1,61 @@
-import React from "react";
+import { useState } from 'react';
+import { EditProductModal } from '../components/EditProductModal';
+import Table from '../components/Table';
+import useFetchProducts from '../hooks/useFetchProducts';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 
-const ProductsPage = () => {
-  return <div></div>;
-};
 
-export default ProductsPage;
+export const ProductsPage = () => {
+  const { products, error, loading } = useFetchProducts();
+  const [selectedProduct, setSelectedProduct] = useState(null);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [localProducts, setLocalProducts] = useState(products);
+
+  const handleEdit = (product) => {
+    setSelectedProduct(product);
+    setIsModalOpen(true);
+  };
+
+  const handleDelete = (id) => {
+    const updatedProducts = localProducts.filter((product) => product.id !== id);
+    setLocalProducts(updatedProducts);
+    localStorage.setItem("products", JSON.stringify(updatedProducts));
+    toast.success("Product Deleted")
```

Comment:
please add a successful word after deleted and make sure to add a style like this
![Screenshot 2024-09-07 120010](https://github.com/user-attachments/assets/39c63576-6155-44b2-8a90-8f3099507cd4)